includes/data-port/models/class-sensei-data-port-lesson-schema.php

```diff
@@ -45,87 +45,110 @@ class Sensei_Data_Port_Lesson_Schema extends Sensei_Data_Port_Schema {
 	public function get_schema() {
 		return [
 			self::COLUMN_ID             => [
+				'name' => __( 'ID', 'sensei-lms' ),
```

Comment:
Not sure that I like the duplication of the column name here. Could we use the `$key` in `restore_from_source_array` and capitalize the first letter instead?

Reply:
I agree with you, but I think the problem with that would be the translation. Do you have some idea to solve this (translate only for the name, and get the text while generating the pot file)?

Reply:
It is a bit strange because for v1, the column names cannot be translated/mapped (we'll expect the headers exactly as-is in English). Since that is the case, we might want to reference the actual English fixed header. We could be explicit in the errors for this (`Error in column "%s": expected...`) or something like that.

Reply:
Good idea! I'll update the message.

Reply:
Updated the text to the new format and used the same pattern to the other columns: https://github.com/Automattic/sensei/pull/3443/commits/c51fce8f330ca44e196b57f9de91ac93980f552c